utils/load-NSF-profiles.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/python
+
+###############################################################################
+# Load NSF xml files into profiles including name, email and often institution
+# NSF data comes from https://www.nsf.gov/awardsearch/download.jsp
+#  -d "/Users/mandler/projects/openreview-scripts/data/2017/"
+###############################################################################
+
+## Import statements
+import argparse
+import os
+import xml.etree.ElementTree as ET
+from openreview import *
+
+## Argument handling
+parser = argparse.ArgumentParser()
+parser.add_argument('-d','--directory', required=True, help = "the full path to the XML files")
+parser.add_argument('--baseurl', help="base url")
+parser.add_argument('--username')
+parser.add_argument('--password')
+
+# catch "not found" errors
+def get_profile(client, value):
+    profile = None
+    try:
+        profile = client.get_profile(value)
+    except openreview.OpenReviewException as e:
+        # throw an error if it is something other than "not found"
+        if e[0][0] != 'Profile not found':
+            print "OpenReviewException: {0}".format(e)
+            raise e
+    return profile
+
+
+def create_profile(client, email, first, last, tilde_id, verbose=False):
```

Comment:
there is a create profile in the library:

https://github.com/iesl/openreview-py/blob/master/openreview/tools.py#L17

It also checks if the profile already exists or not.